vault/resource_identity_entity_alias.go

```diff
@@ -3,6 +3,7 @@ package vault
 import (
 	"context"
 	"fmt"
+	"github.com/hashicorp/vault/api"
```

Comment:
nit: we prefer the that the built-in imports are always grouped together, we can update this after merge however.